build.py

```diff
@@ -98,7 +98,12 @@ def build(config) :
   command += src_files
 
   print('running : %s' % command)
-  print('Build %s' % ('PASSED' if subprocess.call(command) == 0 else 'FAILED'))
+  if subprocess.call(command) != 0:
```

Comment:
This is a NIT but could you put a blank line between 100 and 101 - just like you have in under RUN?